modules/oe_content_tender/config/install/core.entity_form_display.node.oe_tender.default.yml

```diff
@@ -0,0 +1,210 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.oe_tender.body
+    - field.field.node.oe_tender.oe_departments
+    - field.field.node.oe_tender.oe_documents
+    - field.field.node.oe_tender.oe_publication_date
+    - field.field.node.oe_tender.oe_reference
+    - field.field.node.oe_tender.oe_subject
+    - field.field.node.oe_tender.oe_summary
+    - field.field.node.oe_tender.oe_teaser
+    - field.field.node.oe_tender.oe_tender_deadline
+    - field.field.node.oe_tender.oe_tender_opening_date
+    - node.type.oe_tender
+  module:
+    - datetime
+    - link
+    - path
+    - rdf_skos
+    - text
+id: node.oe_tender.default
+targetEntityType: node
+bundle: oe_tender
+mode: default
+content:
+  body:
```

Comment:
Let's order the common fields here like we did in other content types.

Reply:
done